packages/snaps-jest/src/internals/simulation/interface.ts

```diff
@@ -45,33 +46,33 @@ const MAX_FILE_SIZE = 10_000_000; // 10 MB
  */
 export function getInterfaceResponse(
```

Comment:
Out of scope for this PR, but the new dialog type should be supported here

Reply:
Yes I'm going to open a new PR once this one is merged with some support for this new Dialog type in `snaps-jest`